diff -dur zita-alsa-pcmi-0.2.0.orig/libs/Makefile zita-alsa-pcmi-0.2.0/libs/Makefile
--- zita-alsa-pcmi-0.2.0.orig/libs/Makefile	2012-03-20 21:23:07.000000000 +0100
+++ zita-alsa-pcmi-0.2.0/libs/Makefile	2016-10-15 11:35:18.419264378 +0200
@@ -52,7 +52,7 @@
 	install -Dm 644 $(ZITA-ALSA-PCMI_H)   $(DESTDIR)$(PREFIX)/include/$(ZITA-ALSA-PCMI_H)
 	install -Dm 644 $(ZITA-ALSA-PCMI_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/$(ZITA-ALSA-PCMI_MIN)
 	ln -sf $(ZITA-ALSA-PCMI_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/$(ZITA-ALSA-PCMI_SO)
-	ldconfig
+	/sbin/ldconfig -n -N $(DESTDIR)$(PREFIX)/$(LIBDIR)
 
 uninstall:
 	rm -rf $(DESTDIR)$(PREFIX)/include/zita-alsa-pcmi.h
